Sources/Load.php

```diff
@@ -1982,7 +1982,6 @@ function loadTheme($id_theme = 0, $initialize = true)
 		'stats',
 		'notifytopic',
 		'notifyboard',
-		'signup',
```

Comment:
Removed because @MissAllSunday already chose to deal with `signup` another way in 796368c